packages/block-library/src/post-featured-image/block.json

```diff
@@ -24,6 +24,11 @@
 	"usesContext": [ "postId", "postType", "queryId" ],
 	"supports": {
 		"align": [ "left", "right", "center", "wide", "full" ],
+		"color": {
+			"__experimentalDuotone": "img",
+			"text": false,
```

Comment:
Side note, but it's a bit odd we need to toggle these off just to opt-in into duotone. It might make sense to move duotone into a `filters` support in the future.

Reply:
Once we have one more type of filter to justify the new group 🙂

Reply:
I agree with @mtias here. It does look odd and it could justify its own group because you need to add more things and is not just about it's current place under `color`.  Have you considered something similar @nosolosw or @jorgefilipecosta ?